### PR TITLE
be specific when casting boolean-string urlparams

### PR DESCRIPTION
**Pull request by sravfeyn**
https://manage.dimagi.com/default.asp?255822#1347786
@gcapalbo

### CODE REVIEW

File: corehq/apps/userreports/reports/view.py [hunk 1]
```diff
@@ -94,18 +94,27 @@ def get_filter_values(filters, request_dict, user=None):
...
     :return: a dict
     """
     request_dict = json_request(query_dict)
     request_dict['domain'] = domain
+
+    # json.loads casts strings 'true'/'false' to booleans, so undo it
+    for key in string_type_params:
+        val = query_dict.get(key)
+        request_dict[key] = 'true' if val == 'true' else val
+        request_dict[key] = 'false' if val == 'false' else val
```
Comment (gcapalbo) [inline corehq/apps/userreports/reports/view.py]: isn't `'false' if val == 'false' else val` always going to just yield `val`?

Comment (sravfeyn) [inline corehq/apps/userreports/reports/view.py]: Oops, updated

**Review by gcapalbo — APPROVED** [review]
build failure looks related

**Comment by sravfeyn** [comment]
Fixed

**Comment by gcapalbo** [comment]
(looks like build failed on new test)

**Comment by sravfeyn** [comment]
Interesting, I am not able to reproduce this locally

**Comment by sravfeyn** [comment]
Fixed the build.
@gcapalbo

**Pull request merged**